README.md

```diff
@@ -579,32 +579,38 @@ with [environment variables]:
 * `BROWSERSLIST` with browsers queries.
 
    ```sh
-  BROWSERSLIST="> 5%" gulp css
```

Comment:
Why did you make these changes?

Reply:
Could not see the benefit and thought it's cleaner, but I can revert this change

Reply:
I am afraid that many junior developers do not know how to use environment variables.

We can update an example to `npm build` or `webpack` instead of `gulp`.

Reply:
done